drivers/gpio/Kconfig.brcmstb

```diff
@@ -0,0 +1,9 @@
+# Copyright (c) 2024 Junho Lee <junho@tsnlab.com>
+# SPDX-License-Identifier: Apache-2.0
+
+config GPIO_BRCMSTB
+	bool "BRCMSTB GPIO driver"
```

Comment:
It needs a more meaningful description.
For example, 
```
Broadcom Set-top box series GPIO driver
```

Reply:
I changed the description. Thank you for your suggestion.